Sources/SwiftProtobuf/JSONEncodingVisitor.swift

```diff
@@ -280,17 +280,23 @@ internal struct JSONEncodingVisitor: Visitor {
   }
 
   mutating func visitRepeatedEnumField<E: Enum>(value: [E], fieldNumber: Int) throws {
-    let options = self.options
-    let alwaysPrintEnumsAsInts = options.alwaysPrintEnumsAsInts
-    try _visitRepeated(value: value, fieldNumber: fieldNumber) {
-      (encoder: inout JSONEncoder, v: E) throws in
-      if let e = v as? _CustomJSONCodable {
+    if let _ = E.self as? _CustomJSONCodable.Type {
```

Comment:
`if E.self is _CustomJSONCodable.Type {`

Reply:
I basically had that, but switch to this because it is what we have in `visitRepeatedMessageField` right after it.  For the moment, I'll leave it as is so we're consistent.